app/views/posts/index.html.erb

```diff
@@ -1,35 +1,15 @@
-<h2>posts will load Here</h2>
 
-<div class="post-container">
 
-    <div class="post-head-container">
-    <div class="image-container">
-        <h2>Image</div>
-    </div>
+<div class="all-post-container">
+    <h1>Total Posts: <%= @posts.count %></h1>
 
-
-    <div class="text-container">
-        <h3>Post Title</div>
-        <p>Post Content</p>
-    </div>
-    </div>
-
-    <div>
+    <% @posts.each do |post| %>
         <div>
-            <h3>post-1</h3>
-            <p>lorem ipsum</p>
-             <p>comments</p>
-              <p>likes</p>
+        <img src="<%= @user.photo %>" />
+        <h2>Post by <%= post.user.name %></h2>
+        <h3>Title: <%= post.title %></h3>
+        <p>Content: <%= post.text %></p>
         </div>
-        <div class="comment-list">
-        <ul>
-            <li>User-1: comment - 1</li>
-            <li>User-2: comment - 2</li>
-            <li>User-3: comment - 3</li>
-            <li>User-4: comment - 4</li>
-            <li>User-5: comment - 5</li>
-        </ul>
-    
-    </div>
+    <% end %>
```

Comment:
- [ ] Should use post.show_recent_comments to render 5 most recent comments in the app/views/posts/index.html.erb page